templates/pdb.yaml

```diff
@@ -0,0 +1,11 @@
+{{ if .Values.pdb.enabled }}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: terraform-enterprise
```

Comment:
💭 ❓  Is there any need to support annotations or anything like that for the PDB resource? if so we might want to add an empty list value placeholder for it.

Reply:
ah yes I've updated the metadata with namespace, annotations and labels.